### ISSUE TITLE
file Hash calculator

### ISSUE THREAD

**Issue by derekriemer** [karan/Projects#112]
Write an application, which takes in a file name, and a hashing type, I.E. md5, sha1, sha2, crc, bcript, ... and gives you the hash of the given file as a hex string.
For added complexity, take in a GPG key, or a link to one, and verify that the signature on a file is that of the given key.

**Comment by ktulhy-kun** [comment]
Use [hashlib](https://docs.python.org/3/library/hashlib.html), Luke!

**Comment by derekriemer** [comment]
yes, of course. I'm not advocating asking beginners to write cripto libs, which is downright dangerous. I wouldn't attempt writing a cripto lib if my boss asked me to, because i'm simply not qualified.

**Comment by sbelidhe** [comment]
one can use [sphlib](http://www.saphir2.com/sphlib/) to implement file hashing in java. For algorithms like (MD2 , MD5 , SHA-1 , SHA-256 ,  SHA-512 , SHA-384 ) one can refer to [FileHashing](https://github.com/sbelidhe/karangoelprojects/blob/master/FileHashing.java) file written in java for reference
